doc/rados/operations/health-checks.rst

```diff
@@ -1316,35 +1319,41 @@ The location of an OSD can be found with:
 PG_NOT_SCRUBBED
 _______________
 
-One or more PGs has not been scrubbed recently.  PGs are normally scrubbed
-within every configured interval specified by
-:confval:`osd_scrub_max_interval` globally. This
-interval can be overridden on per-pool basis with
-:confval:`scrub_max_interval`. The warning triggers when
-``mon_warn_pg_not_scrubbed_ratio`` percentage of interval has elapsed without a
-scrub since it was due.
+One or more Placement Groups (PGs) have not been scrubbed recently. PGs are
+normally scrubbed within every configured interval determined by
+:confval:`osd_scrub_max_interval` globally. This interval can be overridden on
+per-pool basis by changing the value of the variable
+:confval:`scrub_max_interval`. This health check is raised if a certain
+percentage (determined by ``mon_warn_pg_not_scrubbed_ratio``) of the interval
+has elapsed after the time the scrub was scheduled and no scrub has been
+performed.
+
+PGs will be scrubbed only if they are flagged as ``clean`` (which means that
+they are to be cleaned, and not that they have been examined and found to be
+clean). Misplaced or degraded PGs might not be flagged as ``clean`` (see
+*PG_AVAILABILITY* and *PG_DEGRADED* above).
 
-PGs will not scrub if they are not flagged as *clean*, which may
-happen if they are misplaced or degraded (see *PG_AVAILABILITY* and
-*PG_DEGRADED* above).
+To manually initiate a scrub of a clean PG, run the following command:
 
-You can manually initiate a scrub of a clean PG with::
+.. prompt: bash $
 
-  ceph pg scrub <pgid>
+   ceph pg scrub <pgid>
 
 PG_NOT_DEEP_SCRUBBED
 ____________________
 
-One or more PGs has not been deep scrubbed recently.  PGs are normally
-scrubbed every :confval:`osd_deep_scrub_interval` seconds, and this warning
-triggers when ``mon_warn_pg_not_deep_scrubbed_ratio`` percentage of interval has elapsed
-without a scrub since it was due.
+One or more Placement Groups (PGs) have not been deep scrubbed recently. PGs
+are normally scrubbed every :confval:`osd_deep_scrub_interval` seconds. This
```

Comment:
are periodically scrubbed at most every

There is some randomization, as written this connotes to me that a PG will be scrubbed *exactly* every N seconds.